storybook/src/components/FormFieldDescription/FormFieldDescription.stories.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { FormFieldDescription, FormLabel, TextArea, TextInput } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Components/Forms/Form Field Description',
+  component: FormFieldDescription,
+  args: {
+    children: 'Hint text',
+  },
+} satisfies Meta<typeof FormFieldDescription>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {}
+
+export const AboveFormField: Story = {
+  render: () => (
+    <div style={{ display: 'flex', flexDirection: 'column', gap: '.5rem' }}>
+      <FormLabel htmlFor="about" aria-describedby="aboutDescription">
+        Waar gaat het om?
+      </FormLabel>
+      <FormFieldDescription id="aboutDescription">
+        Typ geen persoonsgegevens in deze omschrijving, dit wordt apart gevraagd
+      </FormFieldDescription>
+      <TextArea id="about" />
+    </div>
+  ),
+}
+
+export const BelowFormField: Story = {
+  render: () => (
+    <div style={{ display: 'flex', flexDirection: 'column', gap: '.5rem' }}>
+      <FormLabel htmlFor="username" aria-describedby="usernameDescription">
+        Kies een gebruikersnaam
+      </FormLabel>
+      <TextInput id="username" />
+      <FormFieldDescription id="usernameDescription">
+        <p>Toegestane tekens</p>
```

Comment:
Not sure if this is what we want. The Paragraph component will bring it's own styling, and I anticipate styling nested `p` and `ul` elements within `amsterdam-form-field-description` is not going to fly.

Reply:
Perhaps it will be acceptable to overrule tokens?

```css
.amsterdam-form-field-description {
  --amsterdam-paragraph-color: var(--amsterdam-form-field-description-color);
}
```

Reply:
We discussed this and decided to keep text in form field descriptions black until we further researched its complexities.